wolox-books/src/services/UserServices.js

```diff
@@ -1,5 +1,5 @@
 import booksApi from '@/config/api'
-import { setValue, removeValue } from './LocalStorageService'
+import { setValue, removeValue, getValue } from './LocalStorageService'
```

Comment:
When we use all the functions in a service or lot of them, we use the `import *`. In this case:

```js
import * as localStorageService from './LocalStorageService'
```

That way, we will have all the functions and then we just access them referencing `localStorageService.<function>`